packages/material-ui/src/Popover/Popover.js

```diff
@@ -117,6 +117,14 @@ const Popover = React.forwardRef(function Popover(props, ref) {
   } = props;
   const paperRef = React.useRef();
 
+
+  const [openState, setOpenState] = React.useState();
+  React.useEffect(()=> {
```

Comment:
This looks like derived state. To avoid tearing we should update the openState during render.